modules/administration-guide/partials/snip_preparing-images-for-a-restricted-environment.adoc

```diff
@@ -0,0 +1,82 @@
+// Included in:
+// modules/administration-guide/pages/installing-che-in-a-restricted-environment.adoc
+// modules/administration-guide/pages/upgrading-che-using-the-cli-management-tool-in-restricted-environment.adoc
+
+
+:my_internal_registry: __<my_internal_registry>__
+
+* The destination OpenShift cluster is ready to operate on a restricted network. See link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-restricted-networks.html[OpenShift documentation - Using Operator Lifecycle Manager on restricted networks].
+
+* An administrative access to the destination OpenShift cluster and the pass:c,a,q[{my_internal_registry}] destination registry.
+
+* The `{prod-cli}` management tool for {prod-short} version {prod-ver} is available on your workstation. See xref:using-the-chectl-management-tool.adoc[].
+
+* The `curl`, `jq`, and `yq` tools are available on your workstation.
+
+* The `opm` tool is available on your workstation. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/opm/cli-opm-install.html[OpenShift documentation - Installing the `opm` CLI].
+
+* Podman is available on your workstation. See link:https://podman.io/getting-started/installation[Podman documentation - Installing Podman].
```

Comment:
I don't see any command using Podman: is it really needed? Isn't skopeo enough?

Reply:
Consider that I usually have an hard time when I need to use podman on my laptop. If that's not needed we should definitely not add it as a pre-requisite.

Reply:
Yes as we borrow it from step 5 in https://docs.openshift.com/container-platform/4.10/operators/admin/olm-managing-custom-catalogs.html#olm-creating-fb-catalog-image_olm-managing-custom-catalogs